internal/provider/resource_application.go

```diff
@@ -961,29 +972,41 @@ func (r *applicationResource) Update(ctx context.Context, req resource.UpdateReq
 	// NOT to update resources, because we want revisions fixed to those
```

Comment:
nit: please update this line of the comments, we're no longer focused solely on revisions.

Reply:
Done